src/parser/warlock/demonology/modules/pets/PetTimelineTab/TabComponent/KeyCastsRow.scss

```diff
@@ -0,0 +1,63 @@
+.key-casts {
+  width: 100%;
+  border-radius: 10px;
+  top: -32px;
+  margin-top: 30px;
+  margin-bottom: -32px;
+  height: 54px;
+  height: calc(54px + var(--levels) * 25px);
```

Comment:
looks like height is specified twice.

Reply:
I think something there takes precedence, or is like a "backup rule". I remember it not working when I removed one of them :D